lib/repl.js

```diff
@@ -289,6 +289,10 @@ function REPLServer(prompt,
     debug('domain error');
     const top = replMap.get(self);
     internalUtil.decorateErrorStack(e);
+    if (e.stack && self.replMode === exports.REPL_MODE_STRICT) {
+      e.stack = e.stack.replace(/(\s+at\s+repl:)(\d+)/,
+                                (_, pre, line) => pre + (parseInt(line) - 1));
```

Comment:
I don't think we need the `parseInt()` here, `"3" - 1` will be faster, and the regex is going to guarantee an integer anyway.